go.mod

```diff
@@ -160,7 +160,7 @@ require (
 	github.com/cpuguy83/go-md2man/v2 v2.0.2 // indirect
 	github.com/curioswitch/go-reassign v0.2.0 // indirect
 	github.com/daixiang0/gci v0.9.1 // indirect
-	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
```

Comment:
Looks like we still have some updates here for some reason - you may have to do a `go mod tidy`

Reply:
Hmm, thought I did, let me just reset this file back to main.